entries/jQuery.isFunction.xml

```diff
@@ -1,14 +1,15 @@
 <?xml version="1.0"?>
-<entry type="method" name="jQuery.isFunction" return="boolean">
+<entry type="method" name="jQuery.isFunction" return="boolean" deprecated="3.3">
   <title>jQuery.isFunction()</title>
   <signature>
     <added>1.2</added>
     <argument name="obj" type="PlainObject">
       <desc>Object to test whether or not it is a function.</desc>
     </argument>
   </signature>
-  <desc>Determine if the argument passed is a JavaScript function object. </desc>
+  <desc>Determine if the argument passed is a JavaScript function object.</desc>
   <longdesc>
+    <p>As of jQuery 3.3, <code>jQuery.isFunction()</code> has been deprecated. In most cases, its use can be replaced by <code>typeof obj === "function"</code>.</p>
```

Comment:
I'd change `typeof obj === "function"` to `typeof x === "function"`; `obj` suggests object is an input and that's not required.

Reply:
Good suggestion. Updating now.